src/cascadia/TerminalConnection/ConhostConnection.cpp

```diff
@@ -12,27 +13,31 @@
 #endif
 
 #include <conpty-universal.h>
+#include "../../types/inc/Utils.hpp"
+
+using namespace ::Microsoft::Console;
 
 namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
 {
-    ConhostConnection::ConhostConnection(hstring const& commandline,
-                                         hstring const& startingDirectory,
-                                        uint32_t initialRows,
-                                        uint32_t initialCols) :
-        _connected{ false },
-        _inPipe{ INVALID_HANDLE_VALUE },
-        _outPipe{ INVALID_HANDLE_VALUE },
-        _signalPipe{ INVALID_HANDLE_VALUE },
-        _outputThreadId{ 0 },
-        _hOutputThread{ INVALID_HANDLE_VALUE },
-        _piConhost{ 0 },
-        _closing{ false }
+    ConhostConnection::ConhostConnection(const hstring& commandline,
+                                         const hstring& startingDirectory,
+                                         uint32_t initialRows,
+                                         uint32_t initialCols,
+                                         const guid& initialGuid) :
+        _initialRows{ initialRows },
+        _initialCols{ initialCols },
+        _commandline{ commandline },
+        _startingDirectory{ startingDirectory },
+        _guid{ initialGuid }
     {
-        _commandline = commandline;
-        _startingDirectory = startingDirectory;
-        _initialRows = initialRows;
-        _initialCols = initialCols;
+        guid zeroGuid{};

```

Comment:
I don't belive `{}` are necessary here (and below), are they doing something other than calling the default constructor?

Reply:
Ensuring that the struct is zero'd which I need to get deterministic comparison, since zero-initialization is not guaranteed unless I call the default constructor as far as I know?